TypeSystem/README.md

```diff
@@ -0,0 +1,88 @@
+# Current Framework Type System
+
+**Current** framework uses its own type system to define data structures and schemas. The main building block is structure, which natively supports reflection and serialization into binary and JSON formats.
```

Comment:
`is a` / `is the` structure.

Reflection for structures is very important. Serialization -- not so much here, imho, since it's a) more complex (think zero-copy), and b) comes after type invariant checking and type system guarantees.